src/allmydata/util/hashutil.py

```diff
@@ -176,10 +176,37 @@ def convergence_hash(k, n, segsize, data, convergence):
     return h.digest()
 
 
-def convergence_hasher(k, n, segsize, convergence):
+def _convergence_hasher_tag(k, n, segsize, convergence):
+    """
+    Create the convergence hashing tag.
+
+    :param int k: Required shares.
+    :param int n: Total shares.
+    :param int segsize: Maximum segment size.
+    :param bytes convergence: The convergence secret.
+
+    :return bytes: The bytestring to use as a tag in the convergence hash.
+    """
     assert isinstance(convergence, bytes)
```

Comment:
Ideally this would raise `ValueError` instead of just asserting

Reply:
I guess, maybe.  Or maybe it should have type annotations and mypy should just tell us there are any broken callers?  Leaving it for now, anyway, since it was here already.